### DISCUSSION TITLE
Custom/Composition template for Pinia stores

### DISCUSSION THREAD

**Discussion by Gerbuuun** [nuxtrdev/nuxtr-vscode#59] [Ideas]
I really like the quick create commands with templates but for Pinia stores it only has the options api version.
I write my stores using the composition api's setup function so it would be cool if I could create a custom template for it or for there to be an option to specify which to generate.

**Comment by adhamfarrag** [comment]
I'm exploring ways to easily migrate customized files/templates across projects/devices without any issue. At the moment Github Gists seem to be the only possible way but still exploring options and tradeoffs.

But I'll add a composition template for the next version. :)

**Comment by adhamfarrag** [comment]
@Gerbuuun, Nuxtr 0.2.4 contains this change. Please, update!
